pkg/api/pod/util.go

```diff
@@ -439,6 +460,32 @@ func dropDisabledFields(
 
 }
 
+// addDisabledFields removes disabled fields from the pod metadata and spec.
+func addDisabledFieldsTo(resetFields *fieldpath.Set) {
+	if !utilfeature.DefaultFeatureGate.Enabled(features.TokenRequestProjection) {
+		// TODO: add the actual fieldpath here, this is just a probably wrong dummy based on
+		// podSpec.Volumes[i].Projected.Sources[j].ServiceAccountToken
```

Comment:
@apelisse can we address fields like this?
The PathElement only knows fieldnames, keys, values, and index but we want to say "every index" or "every key".
Don't think we can do this right now.
This happens in a few cases.

Reply:
Good point, I don't think we have a solution for that with the fieldset. Let me think about it.